### PR TITLE
Fixup of #7381

### PR DESCRIPTION
**Pull request by JohannesLorenz**
Fixup of commit 9c0fc8fc6994b87ece6e706104877b715966af9b

### CODE REVIEW

**Comment by JohannesLorenz** [comment]
@bratpeki please additionally review this, especially make sure this still solves the issue from this comment: https://github.com/LMMS/lmms/pull/7381#issuecomment-2230498800

File: plugins/ZynAddSubFx/ZynAddSubFx.cpp [hunk 1]
```diff
@@ -142,6 +142,11 @@ ZynAddSubFxInstrument::ZynAddSubFxInstrument(
 
 	connect( instrumentTrack()->pitchRangeModel(), SIGNAL( dataChanged() ),
 			this, SLOT( updatePitchRange() ), Qt::DirectConnection );
+
+	// ZynAddSubFX's internal value that LMMS's FREQ knob controls
+	// isn't set properly when the patch is first loaded in,
```
Comment (JohannesLorenz) [inline plugins/ZynAddSubFx/ZynAddSubFx.cpp:147]: "the patch" - not clear what was meant here, need better description

Comment (Rossmaxx) [reply plugins/ZynAddSubFx/ZynAddSubFx.cpp:147]: What about preset?

**Review by LostRobotMusic — APPROVED** [review]

File: plugins/ZynAddSubFx/ZynAddSubFx.cpp (hunk 1 above)
Comment (bratpeki) [reply plugins/ZynAddSubFx/ZynAddSubFx.cpp:147]: LMMS refers to patches as "instruments" (Clear instrument, Instrument bank, etc), you might wanna go with that.

**Review by bratpeki — APPROVED** [review]

File: plugins/ZynAddSubFx/ZynAddSubFx.cpp (hunk 1 above)
Comment (JohannesLorenz) [inline plugins/ZynAddSubFx/ZynAddSubFx.cpp:147]:
```suggestion
	// isn't set properly when the instrument is first loaded in,
```

**Pull request merged**